diff -urEb gdcm-2.8.9.orig/CMakeLists.txt gdcm-2.8.9/CMakeLists.txt
--- gdcm-2.8.9.orig/CMakeLists.txt	2025-09-01 16:44:15.260052498 +0200
+++ gdcm-2.8.9/CMakeLists.txt	2025-09-01 16:45:00.788774640 +0200
@@ -1031,7 +1031,7 @@
   "${GDCM_BINARY_DIR}/Source/Common"
   "${GDCM_SOURCE_DIR}/Source/DataStructureAndEncodingDefinition"
   "${GDCM_SOURCE_DIR}/Source/MediaStorageAndFileFormat"
-  "${GDCM_SOURCE_DIR}/Source/MessageExchangeDefinition"
+  # "${GDCM_SOURCE_DIR}/Source/MessageExchangeDefinition"
   "${GDCM_SOURCE_DIR}/Source/DataDictionary"
   "${GDCM_SOURCE_DIR}/Source/InformationObjectDefinition"
   )
diff -urEb gdcm-2.8.9.orig/Source/CMakeLists.txt gdcm-2.8.9/Source/CMakeLists.txt
--- gdcm-2.8.9.orig/Source/CMakeLists.txt	2025-09-01 16:44:15.246052583 +0200
+++ gdcm-2.8.9/Source/CMakeLists.txt	2025-09-01 16:44:47.887853385 +0200
@@ -6,5 +6,5 @@
   DataStructureAndEncodingDefinition
   InformationObjectDefinition
   MediaStorageAndFileFormat
-  MessageExchangeDefinition
+  # MessageExchangeDefinition
   )
diff -urEb gdcm-2.8.9.orig/Utilities/CMakeLists.txt gdcm-2.8.9/Utilities/CMakeLists.txt
--- gdcm-2.8.9.orig/Utilities/CMakeLists.txt	2025-09-01 16:44:15.256052522 +0200
+++ gdcm-2.8.9/Utilities/CMakeLists.txt	2025-09-01 16:44:41.502892354 +0200
@@ -125,7 +125,7 @@
   set(SOCKETXX_INSTALL_BIN_DIR      ${GDCM_INSTALL_BIN_DIR})
   set(SOCKETXX_INSTALL_LIB_DIR      ${GDCM_INSTALL_LIB_DIR})
   set(SOCKETXX_INSTALL_INCLUDE_DIR  ${GDCM_INSTALL_INCLUDE_DIR}/socketxx)
-  subdirs(socketxx)
+  # subdirs(socketxx)
 endif()
 
 add_subdirectory(gdcmrle)
